src/modules/views/Folder/queries/fetchExtraDrive.ts

```diff
@@ -0,0 +1,83 @@
+import CozyClient, { QueryDefinition } from 'cozy-client'
+import { QueryOptions } from 'cozy-client/types/types'
+
+import logger from 'lib/logger'
+import { buildDriveQuery } from 'modules/queries'
+import {
+  FetchedExtraDriveData,
+  SortOrder,
+  StackClient
+} from 'modules/views/Folder/types'
+
+/**
+ * Asynchronously fetches data from external drives.
+ * @param client - An instance of CozyClient to interact with Cozy's stack.
+ * @returns A Promise resolving to the fetched data as FetchedExtraDriveData.
+ *
+ * Usage:
+ * const data = await fetchExternalDriveData(cozyClient);
+ */
+export const fetchExternalDriveData = async (
```

Comment:
I have the impression that this function is not used anywhere, perhaps a residual from development that needs to be removed

Reply:
Yes it was replaced with the query definition, removing it